packages/manager/src/features/Volumes/VolumeDetails/VolumeEntityDetails/VolumeEntityDetailBody.tsx

```diff
@@ -0,0 +1,147 @@
+
+import type { Volume } from '@linode/api-v4';
+
+interface Props {
+  volume: Volume;
+}
+
+export const VolumeEntityDetailBody = ({ volume }: Props) => {
+  const theme = useTheme();
+  const { data: profile } = useProfile();
+  const { data: regions } = useRegionsQuery();
+
+  const regionLabel =
+    regions?.find((region) => region.id === volume.region)?.label ??
+    volume.region;
+
+  return (
+    <Grid container spacing={1} sx={{ padding: theme.spacingFunction(16) }}>
+      <Grid
+        size={4}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: theme.spacingFunction(8),
+        }}
+      >
+        <Box>
+          <Typography>Status</Typography>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <StatusIcon status={volumeStatusIconMap[volume.status]} />
+            <Typography sx={(theme) => ({ font: theme.font.bold })}>
+              {getFormattedStatus(volume.status)}
+            </Typography>
+          </Box>
+        </Box>
+        <Box>
+          <Typography>Size</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {volume.size} GB
+          </Typography>
+        </Box>
+        <Box>
+          <Typography>Created</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {formatDate(volume.created, {
+              timezone: profile?.timezone,
+            })}
+          </Typography>
+        </Box>
+      </Grid>
+
+      <Grid
+        size={4}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: theme.spacingFunction(8),
+        }}
+      >
+        <Box>
+          <Typography>Volume ID</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {volume.id}
+          </Typography>
+        </Box>
+        <Box>
+          <Typography>Region</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {regionLabel}
+          </Typography>
+        </Box>
+        <Box>
+          <Typography>Volume Label</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {volume.label}
+          </Typography>
+        </Box>
+      </Grid>
+
+      <Grid
+        size={4}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          gap: theme.spacingFunction(8),
+        }}
+      >
+        <Box>
+          <Typography>Attached To</Typography>
+          <Typography sx={(theme) => ({ font: theme.font.bold })}>
+            {volume.linode_id !== null ? (
+              <Link
+                className="link secondaryLink"
+                to={`/linodes/${volume.linode_id}/storage`}
+              >
+                {volume.linode_label}
+              </Link>
+            ) : (
+              <Typography data-qa-unattached>Unattached</Typography>
+            )}
```

Comment:
Seeing a related console error:

<img width="443" height="212" alt="image" src="https://github.com/user-attachments/assets/c1e8c3cf-b049-4356-946c-d14402d859ca" />

(this should also fix the current bolding issue:

<img width="400" height="121" alt="image" src="https://github.com/user-attachments/assets/bc8887e1-8ff5-410f-9bd0-00b89ee057f1" />)


```suggestion
            ) : (
              'Unattached'
            )}
```

Reply:
there's another console error for the tooltip - will create a separate ticket on our end for that